editoast/src/main.rs

```diff
@@ -376,8 +377,36 @@ async fn import_rolling_stock(
             "🍞 Importing rolling stock {}",
             rolling_stock.name.clone().unwrap().bold()
         );
+        let is_electric = match &rolling_stock.effort_curves {
```

Comment:
This check is only done using CLI. We must do the same check when a rolling stock is created or updated using endpoints.

-------

To avoid code duplication we should do this check when deserializing a `RollingStockCommon`. To do so we must implement manually the trait `Deserialize`.

Reply:
done in RollingStockModel